t/ui/18-tests-details.t

```diff
@@ -106,6 +106,7 @@ is($driver->find_element('#user-action a')->get_text(), 'Logged in as Demo', "lo
 $driver->get("/tests/99946");
 $driver->title_is('openQA: opensuse-13.1-DVD-i586-Build0091-textmode@32bit test results', 'tests/99946 followed');
 like($driver->find_element('link[rel=icon]')->get_attribute('href'), qr/logo-passed/, 'favicon is based on job result');
+wait_for_ajax;
```

Comment:
in all your "wait_for_ajax" calls can you now please add a unique test message which I think you also prefer to have. E.g. `wait_for_ajax(msg => 'wait for details loaded before accessing installer_timezone module')`